package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "office-addin-lint check",
     "lint:fix": "office-addin-lint fix",
     "prettier": "office-addin-lint prettier",
-    "start": "office-addin-debugging start manifest.xml",
+    "start": "npm run build & office-addin-debugging start manifest.xml",
```

Comment:
Did we add this to the other templates? I guess it can make sense if you always need to build before starting

Reply:
This is the only place I found that seemed to require it . . . I'm not sure why